inc/container.class.php

```diff
@@ -1165,6 +1165,20 @@ static function validateValues($data, $itemtype, $massiveaction) {
          'plugin_fields_containers_id' => $data['plugin_fields_containers_id']
       ]);
 
+      if (isset($data['items_id'])) {
```

Comment:
Let's say we have a status `discarded` on which we want to set the `discarded_date` mandatory.
If user creates a Computer, `$data['items_id']` would not be set, so we will not apply override rules, so this new Computer can be created using the `discarded` status and an empty `discarded_date`.

Reply:
ping @cconard96

Reply:
The client has said they don't want the overrides to apply for new items.
We need to either remove that one commit and make it a patch specifically for them or add an option on the overrides themselves to indicate if they should apply to new items.

Reply:
> The client has said they don't want the overrides to apply for new items.

Is their field container is located in the main item form or in a dedicated tab ?

Reply:
Their example was in the main form.

Reply:
#543 is introducing this kind of AJAX behaviour. I guess it could be reused here.